src/TraceEvent/Parsers/ClrTraceEventParser.cs

```diff
@@ -191,7 +193,44 @@ public enum Keywords : long
             /// </summary>
             GCHeapSnapshot = GC | GCHeapCollect | GCHeapDump | GCHeapAndTypeNames | Type,
         };
-        public ClrTraceEventParser(TraceEventSource source) : base(source) { }
+        public ClrTraceEventParser(TraceEventSource source) : base(source)
```

Comment:
❓ How do changes like this work with the "TraceParserGen" tool mentioned at the top of this file?